Makefile

```diff
@@ -8,6 +8,10 @@ build:
 	REACT_APP_BASE_URL=$(REACT_APP_BASE_URL) \
 	npm run build
 
-.PHONY: s3_upload
```

Comment:
Now I am seeing that we already have this make command. Why don't we keep this, change the name to `testnet_deploy`, create another one `mainnet_deploy` and use the make instead of having the script directly on github actions?

Reply:
Done